backend/src/main/java/de/bund/digitalservice/ris/caselaw/adapter/database/jpa/DocumentalistDTO.java

```diff
@@ -0,0 +1,42 @@
+package de.bund.digitalservice.ris.caselaw.adapter.database.jpa;
+
+import jakarta.persistence.Column;
+import jakarta.persistence.Entity;
+import jakarta.persistence.GeneratedValue;
+import jakarta.persistence.Id;
+import jakarta.persistence.JoinColumn;
+import jakarta.persistence.ManyToOne;
+import jakarta.persistence.Table;
+import jakarta.validation.constraints.NotBlank;
+import jakarta.validation.constraints.NotNull;
+import jakarta.validation.constraints.Size;
+import java.util.UUID;
+import lombok.AllArgsConstructor;
+import lombok.Builder;
+import lombok.Getter;
+import lombok.NoArgsConstructor;
+import lombok.Setter;
+
+@Getter
+@Setter
+@NoArgsConstructor
+@AllArgsConstructor
+@Builder
+@Entity
+@Table(schema = "incremental_migration", name = "documentalist")
+public class DocumentalistDTO {
+
+  @Id @GeneratedValue private UUID id;
+
+  @Column @NotNull private Integer rank;
+
+  @Column
+  @Size(max = 255)
+  @NotBlank
+  private String value;
+
+  @ManyToOne
+  @JoinColumn(name = "documentation_unit_id")
+  @NotNull
+  private DocumentationUnitDTO documentationUnit;
```

Comment:
Ich hab überlegt, ob es nicht korrekter wäre, das irgendwann(!) als ManyToMany abzubilden. Aktuell behandeln wir jeden `Dokumentar`-Eintrag in jeder DocUnit unabhängig. Also wenn in Dokeinheit A und B `Nina` als Dokumentar steht, dann sind das für uns zwei Ninas, obwohl es wahrscheinlich die gleiche Nina ist (zumindest wenn beide Dokeinheiten von derselben Dokstelle sind) - was denkst du?

Reply:
Noch eine andere Sache: ich glaube wir brauchen die Beziehung in der DTO hier gar nicht, weil die DocumentationUnit ihre Documentalists kennt, oder?

Reply:
Stimmt auf jeden Fall, aktuell sind Dokumentare ja leider nur Strings (also wenn sie von Juris kommen zumindest) und nicht eindeutig erkennbar (und teilweise auch irgendwelche Quatsch Werte). Aber perspektivisch wäre es schon gut das zu ändern. Ich weiß nicht ob das vielleicht sogar auch Teil von dem Verwaltungsdaten Epic ist 👀